spec.html

```diff
@@ -11982,12 +11982,12 @@ <h1>InitializeHostDefinedRealm ( ): either a normal completion containing ~unuse
   <emu-clause id="sec-agents">
     <h1>Agents</h1>
 
-    <p>An <dfn id="agent" variants="agents">agent</dfn> comprises a set of ECMAScript execution contexts, an execution context stack, a running execution context, an <dfn id="agent-record" variants="Agent Records">Agent Record</dfn>, and an <dfn id="executing-thread" variants="executing threads">executing thread</dfn>. Except for the executing thread, the constituents of an agent belong exclusively to that agent.</p>
-    <p>An agent's executing thread executes a job on the agent's execution contexts independently of other agents, except that an executing thread may be used as the executing thread by multiple agents, provided none of the agents sharing the thread have an Agent Record whose [[CanBlock]] field is *true*.</p>
+    <p>An <dfn id="agent" variants="agents">agent</dfn> comprises a set of ECMAScript execution contexts, an execution context stack, a running execution context, an <dfn id="agent-record" variants="Agent Records">Agent Record</dfn>, and an <dfn id="executing-thread" variants="executing threads">executing thread</dfn>. Except for the executing thread, Shared Data Blocks, and specification values related to Shared Data Blocks, the constituents of an agent belong exclusively to that agent.</p>
+    <p>An agent's executing thread executes code on the agent's execution contexts independently of other agents, except that an executing thread may be used as the executing thread by multiple agents, provided none of the agents sharing the thread have an Agent Record whose [[CanBlock]] field is *true*.</p>
     <emu-note>
       <p>Some web browsers share a single executing thread across multiple unrelated tabs of a browser window, for example.</p>
     </emu-note>
-    <p>While an agent's executing thread executes jobs, the agent is the <dfn id="surrounding-agent" variants="surrounding agents">surrounding agent</dfn> for the code in those jobs. The code uses the surrounding agent to access the specification-level execution objects held within the agent: the running execution context, the execution context stack, and the Agent Record's fields.</p>
+    <p>While an agent's executing thread is executing code, the agent is the <dfn id="surrounding-agent" variants="surrounding agents">surrounding agent</dfn> for the code being evaluated. The code uses the surrounding agent to access the specification-level execution objects held within the agent: the running execution context, the execution context stack, and the Agent Record's fields.</p>
```

Comment:
```suggestion
    <p>While an agent's executing thread is executing algorithmic steps, the agent is the <dfn id="surrounding-agent" variants="surrounding agents">surrounding agent</dfn> for those steps. They use the surrounding agent to access the specification-level execution objects held within the agent: the running execution context, the execution context stack, and the Agent Record's fields.</p>
```
(to eliminate anomalous uses of the word "code".)

Reply:
Done.